web/crux-ui/src/models/notification.ts

```diff
@@ -9,19 +9,27 @@ export const NOTIFICATION_EVENT_VALUES = [
 ] as const
 export type NotificationEventType = typeof NOTIFICATION_EVENT_VALUES[number]
 
-export type CreateNotification = {
+export type Notification = {
+  id: string
   name: string
   url: string
   type: NotificationType
   active: boolean
+  creator: string
```

Comment:
CreatedBy, no?

Reply:
For some reason it was called `creator` so I used it when refactoring. Maybe because `createdBy` is the user ID, and creator is the name... I can rename it though.

Reply:
Resolved for now.